migrations/versions/01a8183de4df_added_alcohol_degree_column_for_bar_.py

```diff
@@ -0,0 +1,32 @@
+"""added alcohol degree column for bar items
+
+Revision ID: 01a8183de4df
+Revises: 6272774aaf2b
+Create Date: 2023-10-21 13:11:32.500598
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "01a8183de4df"
+down_revision = "6272774aaf2b"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("bar_item", sa.Column("alcohol_degree", sa.Numeric(precision=4, scale=2), nullable=True))
```

Comment:
du coup avec ma proposition ça ferait

```suggestion
    op.add_column("bar_item", sa.Column("alcohol_degree", sa.Numeric(precision=4, scale=2), nullable=True))
    op.execute("UPDATE bar_item SET alcohol_degree = 5 * is_alcohol::int;")
    op.alter_column("bar_item", "alcohol_degree", nullable=False)
    op.drop_column("bar_item", "is_alcohol")
```
(ptit tricks de cast le bool en int pour faire 5 ou 0, à vréifier que ça marche bien mais normalement ouais, sinon faut taper dans le [CASE WHEN](https://docs.postgresql.fr/10/functions-conditional.html))